units/systemd-firstboot.service

```diff
@@ -15,7 +15,7 @@ ConditionPathIsReadWrite=/etc
 ConditionFirstBoot=yes
 
 DefaultDependencies=no
-After=systemd-remount-fs.service
+After=local-fs-pre.target
```

Comment:
honestly, looking at this, i find this harder to grok then if we'd add an explicit target `rootfs-ready.target` or so. In a way we overload the meaning of local-fs-pre.target here with more than the name suggests.

If I look at this resulting unit file now without knowing the history of this PR I am puzzled: why would we order things after "local-fs-pre.target"? this is not a local mount, so this seems wrong.

I think adding one more target is OK if it makes the unit files more readable and obvious, but this change here really does not.